src/tests/0000ztrips.js

```diff
@@ -138,4 +158,53 @@ describe('trips tests', () => {
         done();
       });
   });
+  it('manager should get request made by his/her own direct', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests')
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('object');
+        chai.expect(res.body.message).to.eq('Trips requested by your direct reports');
+        done();
+      });
+  });
+  it('manager should get requests on the selected page', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests?page=1')
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('object');
+        chai.expect(res.body.message).to.eq('Trips requested by your direct reports');
+        done();
+      });
+  });
+  it('should get error when a page is requesting dont have data', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests?page=2')
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(404);
+        res.body.should.be.an('object');
+        chai.expect(res.body.error).to.eq('No trip requests on this page');
+
+        done();
+      });
+  });
+  it('paginating tests', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests?page=-1')
+      .set('token', `Bearer ${token2}`)
+      .end((err, res) => {
+        res.should.have.status(200);
+        res.body.should.be.an('object');
+        done();
+      });
+  });
+  it('manager should not get request when he is not a manager', (done) => {
+    chai.request(app).get('/api/v1/trip/trip-requests')
+      .set('token', `Bearer ${token3}`)
+      .end((err, res) => {
+        res.should.have.status(401);
+        res.body.should.be.an('object');
```

Comment:
@mujohn26 Testing, if the response is an object, will always pass, you want instead test both the properties and their values of the response object. You want to test things that might be changed in the code you wrote. Having this in mind, a response will always be an object, but maybe someone might change the response message or any other response property. Kindly update elsewhere you're testing the type of the response object.

Reply:
I resolved it.